CMD/Spritz.cs

```diff
@@ -54,6 +54,31 @@ public static void Main(string[] args)
             #endregion STAR Fusion Testing
 
             #region Proteoform Database Engine
+            //lncRNA workflow
+            if (options.lncRNAWorkflow)
+            {
+                if (options.Fastq2 != null && options.Fastq1.Count(x => x == ',') != options.Fastq2.Count(x => x == ','))
+                    return;
+
+                string[] fastqs1 = options.Fastq1.Split(',');
+                List<string[]> fastqsSeparated = options.Fastq2 == null ?
+                    fastqs1.Select(x => new string[] { x }).ToList() :
+                    fastqs1.Select(x => new string[] { x, options.Fastq2.Split(',')[fastqs1.ToList().IndexOf(x)] }).ToList();
+
+                LncRNADiscoveryWF.RunLncRNADiscoveryFromFastq(
```

Comment:
Could we name this class `LncRNADiscoveryFlow` for consistency with the other workflows? (Flow is a cool, short word. That's why I like it.)